File: Programación Orientada a Objetos (PDOO)/Napakalaki(Java)/src/napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author joseperoman
 */
public class Prize {
    private int treasures;
    private int level;
    
    public Prize(int trss, int lvl){
        treasures = trss;
        level = lvl;
    }
    
    public int getTreasures(){
        return treasures;
    }
    
    public int getLevel(){
        return level;
    }
    
    public String toString(){
        return " \n\tTreasures = " + Integer.toString(treasures) + "; \n\tLevel = " + Integer.toString(level);
    }
}

File: Programación Orientada a Objetos (PDOO)/Napakalaki(Ruby)/pkg/pruebaNapakalaki-0.0.1/lib/prize.rb
#encoding: utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "bad_consequence.rb"
require_relative "monster.rb"
require_relative "treasure_kind.rb"
module NapakalakiGame
class Prize
  attr_accessor:treasures
  attr_accessor:level
  
  def initialize(trs,lvl)
    @treasures = trs
    @level = lvl
  end
  
  def self.newPrize(trs,lvl)
    new(trs,lvl)
  end

  def to_s
    "Tesoros ganados: #{@treasures}\n Niveles ganados: #{@level}"
  end
end
end